MonoGame.Framework/Graphics/SurfaceFormat.cs

```diff
@@ -104,6 +104,32 @@ public enum SurfaceFormat
         /// For compatibility with WPF D3DImage.
         /// </summary>
         Bgra32 = 21,    // B8G8R8A8    
+
+        /// <summary>
+        /// Unsigned 32-bit RGBA sRGB pixel format that supports 8 bits per channel.
+        /// </summary>
+        ColorSRgb = 30,
+        /// <summary>
+        /// Unsigned 32-bit sRGB pixel format that supports 8 bits per channel. 8 bits are unused.
+        /// </summary>
+        Bgr32SRgb = 31,
+        /// <summary>
+        /// Unsigned 32-bit sRGB pixel format that supports 8 bits per channel.
+        /// </summary>
+        Bgra32SRgb = 32,
+        /// <summary>
+        /// DXT1. sRGB texture format with compression. Surface dimensions must be a multiple of 4.
+        /// </summary>
+        Dxt1SRgb = 33,
+        /// <summary>
+        /// DXT3. sRGB texture format with compression. Surface dimensions must be a multiple of 4.
+        /// </summary>
+        Dxt3SRgb = 34,
+        /// <summary>
+        /// DXT5. sRGB texture format with compression. Surface dimensions must be a multiple of 4.
+        /// </summary>
+        Dxt5SRgb = 35,
+

```

Comment:
No, at least according to SharpDX - these are all but one of the SRgb formats that the SharpDX `Format` enum exposes. (The only other one is `BC7_UNorm_SRgb`, but we don't support the non-SRgb version of that.)